addons/account/static/src/components/many2many_tags_banks/many2many_tags_banks.xml

```diff
@@ -1,8 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
     <t t-name="FieldMany2ManyTagsBanksTagsList" t-inherit="web.TagsList" t-inherit-mode="primary">
-        <xpath expr="//div[hasclass('o_tag_badge_text')]" position="after">
-            <i t-if="tag.allowOutPayment" class="ms-1 fa fa-unlock"/>
+        <xpath expr="//span[hasclass('o_tag')]" position="attributes">
+            <attribute name="t-att-data-tooltip" add="` (${trust})`" separator=" + "/>
+            <attribute name="t-att-aria-label" add="` (${trust})`" separator=" + "/>
+        </xpath>
```

Comment:
Can't we just remove this?
I find the tooltip on the icon enough, but this is still not easy to translate I think. Does it behave nicely in both right-to-left and left-to-right languages?

On the icons it is easy to add a static tooltip on each line.